install_sct

```diff
@@ -744,3 +745,5 @@ else
 fi
 
 ) 2>&1 | tee install_sct_log.txt
+
+popd
```

Comment:
By default, `pushd` and `popd` echo the directory stack back to `stdout`. This looks a bit odd without the extra context, I think? So, we might want to consider:

1. Suppressing the messages using `> /dev/null`, and/or
2. Adding a more informative message, such as "Temporarily changing directories to <>" and "Returning to the original directory..."

Reply:
Good point, I added `>/dev/null` to both.